bigquery_etl/cli/query.py

```diff
@@ -204,7 +217,20 @@ def create(name, sql_dir, project_id, owner, init):
 
 
 @query.command(
-    help="Schedule an existing query",
+    help="""Schedule an existing query
+
+    Examples:
```

Comment:
It looks like we have a mix of some of these multiline strings having content indented all the way left, and sometimes indented at the same level as the `help` kwarg. My preference would be to consistently indent like is done here. https://docs.python.org/3/library/textwrap.html may be useful if the indentation is causing problems.